### ISSUE TITLE
Flows To Use Service Principals Not Service Accounts

### ISSUE THREAD

**Issue by NPrice99** [microsoft/coe-starter-kit#465]
Make whole of CoE be capable of using Service Principals rather than Service Accounts as service accounts are insecure.

**Describe the solution you'd like**
Whole of CoE to use Service Accounts.

**Describe alternatives you've considered**
None

**Additional context**
A number of institutions ban Service Accounts as they are insecure and only use Service Principals.

I have, on my own personal tenancy, have a number of flows using service Principals (but there are so many of them it is taking time to convert them and everytime there is a new release I will have to convert them again.

**Comment by manuelap-msft** [comment]
At the moment, only a limited number of connectors (mostly Azure ones) support using Service Principals from flow. Whilst we can have this item on our backlog, we will depend on the product to make Service Principals available for the connectors we use - such as Power Apps for Admins etc. Our hands are tied until this capability becomes available in the product.

**Comment by MarinaHadzi** [comment]
Hi @manuelap-msft, can we set up CoE now with Service Principal account on Teams or on Production or both?

**Comment by russrimm** [comment]
@MarinaHadzi I don't believe all connectors the CoE uses support SPNs, yet.  When they do, I wonder if the 25,000 base request limit implied on service principals since they're non-licensed will cause a problem in larger environments?

**Comment by mohamrizwa** [comment]
We noticed that this issue has been pending for quite some time. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved.
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue.
Thank you for your understanding and cooperation.
